package.json

```diff
@@ -27,20 +27,20 @@
 		"test:visualtest": "npm run backstopjs -- --storybook-host=./dist/",
 		"test:visualapprove": "backstop approve --config=tests/backstop/config.js",
 		"test:analyze": "npm run build:production -- --env.ANALYZE",
-		"test:js": "wp-scripts test-unit-js --config=tests/js/jest.config.js",
-		"test:js:watch": "wp-scripts test-unit-js --config=tests/js/jest.config.js --watch",
+		"test:js": "cross-env BABEL_ENV=test NODE_ENV=test jest --config=tests/js/jest.config.js --passWithNoTests",
```

Comment:
❤️  for using `cross-env`

Reply:
Yup, it's a good shout! Straight from the IB that one. :)